dist/api_data.py

```diff
@@ -1060,7 +1060,7 @@ def __ge__(self, other):
         chunk_cache_configuration_common + [
         Config('capacity', '10GB', r'''
             maximum memory or storage to use for the chunk cache''',
-            min='0', max='100TB'),
+            min='512KB', max='100TB'),
```

Comment:
We have changed the default to 512KB. Thoughts?

Reply:
This is the minimum that changed, right? Not the default? I think having a non-zero minimum documented makes sense, the code checks for that at the moment anyway.

Reply:
Yes, it is for the minimum.